apps/frontend/components/repository/Workflow/Sidebar/SidebarBody.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div>
+    <RichTextEditor
+      :model-value="status.description"
+      class="mb-2"
+      data-testid="description-input"
+      label="Description"
+      variant="outlined"
+      @change="updateStatus('description', $event)"
+    />
+    <SelectStatus
+      :items="workflow.statuses"
+      :model-value="status.status"
+      data-testid="status-input"
+      label="Status"
+      variant="outlined"
+      @update:model-value="updateStatus('status', $event)"
+    />
+    <VSelect
+      :items="users"
+      :model-value="status.assigneeId"
+      class="my-2"
+      data-testid="assignee-input"
+      item-title="label"
+      item-value="id"
+      label="Assignee"
+      placeholder="Click to set assignee"
+      variant="outlined"
+      clearable
+      @update:model-value="updateStatus('assigneeId', $event)"
+    >
+      <template #selection="{ item }">
+        <VAvatar :image="item.raw.imgUrl" class="mr-4" size="26" />
+        {{ item.title }}
+      </template>
+      <template #item="{ item, props: selectProps }">
+        <VListItem v-bind="selectProps">
+          <template #prepend>
+            <VAvatar :image="item.raw.imgUrl" size="26" />
+          </template>
+        </VListItem>
+      </template>
+    </VSelect>
+    <SelectPriority
+      :items="workflowConfig.priorities"
+      :model-value="status.priority"
+      class="mb-2"
+      data-testid="priority-input"
+      label="Priority"
+      variant="outlined"
+      @update:model-value="updateStatus('priority', $event)"
+    />
+    <VDateInput
+      :model-value="dueDate && new Date(dueDate)"
+      data-testid="date-input"
```

Comment:
consider using a bit more specific bem like ids for testid e.g. componentName_elementName